# Introduction
_Data scientist and engineer with a Ph.D. in physics and extensive experience in biomedical data science, bioinformatics, and computational biology. Specializes in complex data curation, bioinformatics pipeline development, and cloud-based computational infrastructure. Passionate about leveraging data to address real-world challenges._

_I will be most effective on a team that embraces neurodiversity and can leverage unique cognitive styles._

# Skills

**Data Science & Engineering:** ETL, Data harmonization, SQL, Cloud (GCP), Docker, HPC (SLURM), Jupyter Notebooks, R-tidyverse, ggplot, matplotlib, seaborn.

**Bioinformatics & Genomics:** DNA/RNA sequencing (RNA-Seq, ChIP-Seq), Variant calling, Microbiome analysis, Transcriptomics, DepMap, ClinVar, KEGG.

**Programming & Development:** *Expert***:** R, Bash; *Strong***:** Python, SQL, HTML/CSS/JS; *Familiar***:** Matlab, C++, Java.

**Healthcare Analytics:** ICD-10, CPT, HCPCS, CMS, Medicaid claims, Quality Metrics (HEDIS).

**Linux & Systems Administration:** Server configuration, Web servers, Package management, SLURM, HTCondor.

# Accomplishments
* Launched a funded nonprofit organization creating solutions for Autistic adults in Oregon.
* Created and maintained pipelines for ETL and quality metrics for Medicaid claims data that run on Slurm HPC cluster.  
* Launched a corporate-academic collaborative research project by curating multiple large-scale public genomic data sets into a graph database suitable for machine learning, oncology, and oncological drug repurposing.  
* Created reproducible, templated pipelines that perform multi-stage statistical and quantitative analysis and present data using text, tables, and graphical visualizations.  
* Created and coded real-time dashboard plots for a web-based lab notebook for targeted cancer treatment trials.  
* Lead author on theoretical and computational methods publications physics, biophysics, computational biology, and physical chemistry, and co-authored papers using bioinformatic analysis and image processing pipelines.

# Employment
**11/01/2024-Present:** Founder/Directory (Autism All Grown Up [https://aagu.org](https://aagu.org))
* Founded and launched Autism All Grown Up (AAGU), an Autistic-run nonprofit in Oregon dedicated to empowering the Autistic community by addressing systemic gaps and societal barriers. Our mission is to create spaces, systems, and resources that reflect and respect the lived experiences of Autistic adultsâ€”especially those who fall through the cracks of existing services.
* Raised over $60k in grant funding between December 2024 and March 2025\.
* Piloting a funded first-of-its-kind data collection project on the lived experiences of Autistic adults in Oregon.
* Launched other programs with community partners.
* Providing jobs for Autistic adults with professional skills.

**06/01/2022-12/31/2023:** Biomedical Data Science Contractor (Katana Graph, Providence Health and Services, Oregon Health and Science University, Portland State University)
* Data Engineering and Curation (curation, sourcing, cleaning, normalization, harmonization, ETL). Cloud database design and management.
* Statistical modeling and analysis using R, Python, and Bash
* Linux command line operations and Bash shell scripting
* Bioinformatics: pipeline development, analysis, variant calling, RNA-Seq, ChIP-Seq, Proteomics, next-generation sequencing (NGS), transcriptomics, etc.
* Using cloud computing (Google Cloud Platform) technology such as virtual machines and databases
* Genomics subject matter expert for using publically available oncology datasets and databases for cancer drug discovery.

**11/21/2022-9/30/2023:** Senior Research Engineer, Center for Health Systems Effectiveness (OHSU)
* Simplified and accelerated pipelines for staging CMS health records running on a high-performance computing (HPC) cluster.
* Simplified and accelerated analytical pipelines for Medicaid health record analysis (HEDIS Quality Measures) running on a high-performance computing (HPC) cluster.
* Led improvements in computing resources, programming, and analytical workflows.
* Created education materials to improve computation methods.

**08/24/2020-06/08/2022:** Bioinformatics Scientist, Providence Portland Medical Center, Portland OR
* Launched a business-academic collaborative research project integrating multiple large-scale public genomic data sets (DepMap, GDSC, etc.)  into a graph database for machine learning, cancer drug repurposing, and genomic and functional analysis.
* Data curation and data engineering. Combined clinical genomic data with public genomic data, including next-generation sequencing (NGS) transcriptomics multi-cell RNA-Seq, DNA-Seq, WES, WGS, and panel sequencing to help predict disease risk.
* Maintained and managed multiple BigQuery (Google Cloud Platform) databases through scripted data, acquisition, cleaning, harmonization, and ETL.
* Incorporated multiple data formats, including databases, FASTA files, JSON, CSV, and TSV
* Helped develop dockerized genomic analysis pipelines with Linux Bash scripting, R, Python, and Docker
* Statistical modeling and analysis using R, Python, and Rmarkdown
* Unix/Linux command line operations and Bash shell scripting

**08/15/2018-03/31/2020:** Computational Biologist, Casey Eye Institute, Oregon Health and Science University, Portland OR
* Processed next-generation sequencing (NGS) microbiome data. Performed differential abundance analysis and functional analysis (KEGG).
* Statistical analysis for a human subjects study
* Created a reproducible pipeline using R, Rmarkdown, and Bash for statistical analysis of microbiome data with Bioconductor tools and other packages
* Performed analysis on a high-performance computing (HPC) cluster using SLURM
* Incorporated multiple data formats, including databases, FASTA files, JSON, CSV, and TSV
* Contributed to publications

**11/15/2015-04/01/2017:** Computational Biologist, Bioinformatics Core, Oregon Health and Science University, Portland OR
* Created reproducible, templated pipelines using Bash and Python scripting,  standard command-line bioinformatics tools, and Bioconductor packages for next-generation sequencing (NGS) transcriptomics analysis (ChIP-Seq, RNA-Seq) on a high-performance computing (HPC) cluster using HT Condor and SLURM.
* Accessed and processed public data sets such as the human genome, mouse genome.
* Incorporated multiple data formats, including databases, FASTA files, JSON, CSV, and TSV
* Designed and coded real-time dashboard plots for a web-based lab notebook for targeted cancer treatment trials (Javascript, Plotly)
* Contributed to three publications by providing next-generation sequence analysis and increasing an image processing pipeline throughput by 100x.
* Coordinated and interfaced between multiple research groups and clients.
* Planned and led workshops on computational skills and methods.

**02/10/2009-12/31/2012:** Postdoctoral Research Assistant, Pacific Northwest National Laboratory, Richland, WA
* Created a novel stochastic mathematical and computational model for diffusion, incorporating density fluctuations using Matlab and Bash on high-performance clusters. 
* Created a novel mathematical and computational model to interface a stochastic surface reaction to continuous field diffusion.  
* Created a novel analytical solution to a stochastic reaction-diffusion equation that captures the anomalous reaction rate. 

# Education
**Graduate coursework in Biostatistics.** Oregon Health and Science University. *Linear Models. Design and Analysis of Surveys. Categorical Data Analysis.* Master's degree eventually.

**AAS (August 2015\) (Software Development).** Columbia Basin College, Pasco, WA

**PhD (May 2005\) (Theoretical Physics, Math minor).** Indiana University, Bloomington, IN **Award**: Outstanding graduate student in research.

**BS (June 1998\) (Physics).** The Evergreen State College, Olympia, WA

# Awards
Outstanding Graduate Student in Research (Indiana University, May 2005)


