# Introduction

_I'm a Ph.D. physicist turned biomedical data scientist. I have several exceptional strengths: a keen and discerning scientific mind, the ability to swiftly identify the essence of a task, question, or problem, then assimilate the background and context, wizardry with data engineering, and a tenacious drive for consistency, accuracy, and efficiency._

_If you are open to hiring someone who can be respectfully insubordinate but is honest, loyal, and endowed with remarkable ingenuity and perseverance, I believe you will be glad you gave me a chance to work for you._

# Accomplishments
- Launched a corporate-academic collaborative research project integrating multiple large-scale public genomic 
  data sets into a graph database suitable for machine learning.
  
- Created reproducible, templated pipelines for multi-stage statistical analysis of next-generation sequencing data.

- Created and coded real-time dashboard plots for a web-based lab notebook for targeted cancer treatment trials.

- Lead author on theoretical and computational methods publications physics, biophysics, computational biology, and 
  physical chemistry, and co-authored papers using bioinformatic analysis and image processing pipelines.

- Designed and maintained databases at the Hanford safety library.

- Promotes reproducible research with literate programming, notebooks, self-documenting code, version control, and strict 
 data provenance by example and through mentoring and teaching.
 
 
# Skills
**Data Science:** EDA. Data cleanup, normalization, and harmonization. Database systems. Cloud computing. Multi-language pipelines. 
  Jupyter and R Notebooks. R-Tidyverse. Conda. GGplot. Matplotlib. Seaborn. Docker.

**Statistics/Biostatistics:** Inference and hypothesis testing; Regression and linear models; Model selection, diagnostics, 
  and post-hoc analysis; Univariate statistics (parametric and non-parametric); Ordinationl Diversity analysis; Multiple 
  comparisons; Differential abundance analysis. Categorical/discrete variables. Survey design. Genetic risk. Functional analysis.

**Bioinformatics:** Microbiome analysis, RNA-Seq, ChIP-Seq, Variant calling.

**Cloud Computing:** Google Cloud Platform; HPC cluster job submission; Docker; Pipelines.

**Programming Languages:** <span style="text-decoration:underline;">Expert</span>: R, Bash/Shell (awk, sed, regex, etc.). 
  <span style="text-decoration:underline;">Strong</span>: Python, Git, SQL, /CSS/JS/JQuery. <span style="text-decoration:underline;">
  Rusty</span>: Matlab. <span style="text-decoration:underline;">Knowledgeable</span>: C++, Java, Visual Basic, ColdFusion, PHP.

**Linux**: Server setup. Package management. Users/Groups/Permissions/ACLs. Web server configuration. Drive mapping. Cluster 
  submission (SLURM, HTCondor).

**Basic Web Development:** HTML/CSS/JS, JQuery, Responsive design, flexbox, bootstrap, Wordpress, SVG, AJAX.

**Outreach, education, mentoring:** Planning and leading workshops. Teaching. Giving talks. Creating demonstrations. Mentoring 
  research students. Community outreach.


# Employment
**06/01/2022â€”Present:** Biomedical Data Science Contractor

* Data Engineering and Curation (curation, sourcing, cleaning, normalization, harmonization, ETL). Cloud database design and management.
* Statistical modeling and analysis using R, Python, and Bash
* Linux command line operations and Bash shell scripting
* Bioinformatics: pipeline development, analysis, variant calling, RNA-Seq, ChIP-Seq, Proteomics, next-generation sequencing (NGS), transcriptomics, etc.
* Using cloud computing (Google Cloud Platform) technology such as virtual machines and databases
* Genomics subject matter expert for using publically available oncology datasets and databases for cancer drug discovery.

**11/21/2022-9/30/2023:** Senior Research Engineer, Center for Health Systems Effectiveness (OHSU)

* Simplified and accelerated pipelines for staging Medicaid health records running on a high-performance computing (HPC) cluster.
* Simplified and accelerated analytical pipelines for Medicaid health record analysis (HEDIS Quality Measures) running on a high-performance computing (HPC) cluster.
* Led improvements in computing resources, programming, and analytical workflows.
* Created education materials to improve computation methods.

**08/24/2020-06/08/2022:** Bioinformatics Scientist, Providence Portland Medical Center, Portland OR

* Launched a business-academic collaborative research project integrating multiple large-scale public genomic data sets (DepMap, GDSC, etc.)  into a graph database for machine learning, cancer drug repurposing, and genomic and functional analysis.
* Data curation and data engineering. Combined clinical genomic data with public genomic data, including next-generation sequencing (NGS) transcriptomics multi-cell RNA-Seq, DNA-Seq, WES, WGS, and panel sequencing to help predict disease risk.
* Maintained and managed multiple BigQuery (Google Cloud Platform) databases through scripted data, acquisition, cleaning, harmonization, and ETL.
* Incorporated multiple data formats, including databases, FASTA files, JSON, CSV, and TSV
* Helped develop dockerized genomic analysis pipelines with Linux Bash scripting, R, Python, and Docker
* Statistical modeling and analysis using R, Python, and Rmarkdown
* Unix/Linux command line operations and Bash shell scripting

**08/15/2018-03/31/2020:** Computational Biologist, Casey Eye Institute, Oregon Health and Science University, Portland OR

* Processed next-generation sequencing (NGS) microbiome data. Performed differential abundance analysis and functional analysis (KEGG).
* Statistical analysis for a human subjects study
* Created a reproducible pipeline using R, Rmarkdown, and Bash for statistical analysis of microbiome data with Bioconductor tools and other packages
* Performed analysis on a high-performance computing (HPC) cluster using SLURM
* Incorporated multiple data formats, including databases, FASTA files, JSON, CSV, and TSV
* Contributed to publications

**11/15/2015-04/01/2017:** Computational Biologist, Bioinformatics Core, Oregon Health and Science University, Portland OR

* Created reproducible, templated pipelines using Bash and Python scripting,  standard command-line bioinformatics tools, and Bioconductor packages for next-generation sequencing (NGS) transcriptomics analysis (ChIP-Seq, RNA-Seq) on a high-performance computing (HPC) cluster using HT Condor and SLURM.
* Accessed and processed public data sets such as the human genome, mouse genome.
* Incorporated multiple data formats, including databases, FASTA files, JSON, CSV, and TSV
* Designed and coded real-time dashboard plots for a web-based lab notebook for targeted cancer treatment trials (Javascript, Plotly)
* Contributed to three publications by providing next-generation sequence analysis and increasing an image processing pipeline throughput by 100x.
* Coordinated and interfaced between multiple research groups and clients.
* Planned and led workshops on computational skills and methods.

**07/2014-11/2015:** IT Specialist, HAMMER Volpentest Federal Training Center, Richland WA

* Database design and implementation (MS Access and MS Excel)
* Library information system design, testing, and implementation
* Coldfusion web application programming

**01/2013-01/2014:** Research Scientist, Washington State University, Richland, WA

* Multilevel, cell-based modeling of epidermis growth

**01/2012-05/2012, 09/2012-12/2012:** Adjunct Physics Instructor, Washington State University, Richland, WA

* Taught lecture and lab section for calculus-based introductory physics for undergraduates.

**02/2009-12/2012:** Postdoctoral Research Assistant, Pacific Northwest National Laboratory, Richland, WA

* Created a novel stochastic mathematical and computational model for diffusion, incorporating density fluctuations using Matlab and Bash on high-performance clusters.
* Created a novel mathematical and computational model to interface a stochastic surface reaction to continuous field diffusion.
* Created a novel analytical solution to a stochastic reaction-diffusion equation that captures the anomalous reaction rate. 

**09/2006-01/2008:** Physics Lecturer, Indiana University, Bloomington IN

* Taught lecture sections for undergraduate physics classes

**05/2005-05/2008:** Postdoctoral Research Associate, Biocomplexity Institute, Bloomington IN

* Created a multi-level, cell-based computational model of chicken gastrulation that elucidated the role of both physical cell-cell interactions and signaling proteins. (Python and Bash)
* Helped improve and refine the Glazier-Graner-Hogoweg agent-based computational morphogenesis model.
* Contributed to papers and book chapters.

**05/2001-05/2005:** Graduate Research Assistant, Indiana University, Bloomington IN

* Thesis work on theoretical and computational modeling of actin-based bacterial motility (Matlab and Fortran)
* Worked in the free physics tutoring clinic
* Participated in physics community outreach and education

**09/1998-5/2001:** Associate Instructor, Indiana University, Bloomington IN

* Taught lab classes for physics undergraduate students
* Worked in the free physics tutoring clinic
* Participated in physics community outreach and education

# Education
**AAS (2015) (_Software Development_).**  Columbia Basin College, Pasco, WA

**PhD (2005) (_Theoretical Physics, Math minor_).**  Indiana University, Bloomington, IN **Award**: _Outstanding graduate student in research._

**BS (1998) (_Physics_).**  The Evergreen State College, Olympia, WA  


# Selected Publications  
**_A diet rich in fermentable fiber promotes robust changes in the intestinal microbiota, mitigates intestinal permeability, and attenuates autoimmune uveitis_** Yukiko K. Nakamura, et al. [Nature Scientific Reports volume 13, Article number: 10806 (2023)](https://www.nature.com/articles/s41598-023-37062-8)

**_Swept-source optical coherence tomography objective composite activity score for uveitis._** LlorenÃ§ et al. [ Acta Ophthalmol](https://onlinelibrary.wiley.com/doi/10.1111/aos.14739), 09 January 2021

**_Maintenance of MYC expression promotes de novo resistance to BET bromodomain inhibition in castration-resistant prostate cancer_.** Coleman et al.[ Nature Scientific Reports, Vol. 8, 3823 (2019)](https://www.nature.com/articles/s41598-019-40518-5)

**_BET Bromodomain Inhibition Blocks the Function of a Critical AR-Independent Master Regulator Network in Lethal Prostate Cancer._** Coleman et al.[ Nature: Oncogene, 17 April 2019](https://www.nature.com/articles/s41388-019-0815-5)

**_Quantitative Multiplex Immunohistochemistry Reveals Myeloid-Inflamed Tumor-Immune Complexity Associated with Poor Prognosis._** Tsujikawa et al.[ Cell Reports, Volume 19, Issue 1, p203â€“217, 4 April 2017.](https://www.cell.com/cell-reports/abstract/S2211-1247%2817%2930383-2%3E)

**_Effect of Nonlinearity in Hybrid KMC-Continuum models._** Ariel Balter, Guang Lin, Alexandre M. Tartakovsky.[ PhysRevE.85.016707.](https://journals.aps.org/pre/)abstract/10.1103/PhysRevE.85.016707)

**_Multinomial Diffusion Equation._** Ariel Balter, Alexandre Tartakovsky.[ PhysRevE.83.061143](https://journals.aps.org/pre/abstract/10.1103/PhysRevE.83.061143)

**_Modeling Gastrulation in the Chick Embryo: Formation of the Primitive Streak._** Bakhtier Vasiev, Ariel Balter, Benji Zaitlen, James A. Glazier, Mark Chaplain, Cornelis J. Weijer. PLoS One. 2010; 5(5): e10571.

**_Probing Soap-Film Friction with Two-Phase Foam Flow_**. Ariel Balter, Rebecca Perry, James A. Glazier. Philosophical Magazine Letters Vol. 8 Issue 9,2008 p. 679 - 691.

**_Hydrodynamic Stability of Helical Growth at Low Reynolds Number._** Ariel Balter, Jay X. Tang. Phys. Rev. E 71, 051912 (2005).


# STEM Outreach

**2016, OHSU Data Jamborees:** Organized educational events with speakers and workshops on data analysis skills and computational resources. Led workshops attended by 100+ students, postdocs, and faculty.

**2007-2008, Wonderlab:** Helped design and build exhibits for a childrenâ€™s science museum in Bloomington (IN).

**1998-2007, IU Physics Department Open House:** Designed and built new displays and activities. Participated in setup and docenting.


# Bio
I returned to school at age 28 to complete a bachelorâ€™s degree; Ph.D. at age 37. Before that, I worked in restaurants, freelance writing, and primarily in carpentry/woodworking. Hobbies include cooking, baking artisan bread, gardening/preserving, building things. I play guitar and mandolin ok. Iâ€™m nice (usually).
